docs/CODEOWNERS

```diff
@@ -0,0 +1,144 @@
+# Archipelago World Code Owners / Maintainers File
+#
+# This file is used to notate who the current "owners" or "maintainers" of any currently merged world folder. For any
```

Comment:
"file" could be changed to "document" to be more formal and match later instances. "notate" is usually used with music or shorthand writing, changing it to "note" or "list" would be better. You can also remove "who" from the sentence as it makes it grammatically incorrect. "content of these worlds" should be "the contents of these worlds"